tests/system/pages/_elements/LoginPageElements.ts

```diff
@@ -0,0 +1,13 @@
+import { BasePageElements } from './BasePageElements';
+
+export class LoginPageElements implements BasePageElements {
```

Comment:
Plain js object instead of class?

Reply:
In that case I will force to implement additional interface for LoginPageElements in order to get easy access and checks for parameters, currently it looks like overenginering, but when project will grow up, it will be tricky to manage different pages with all elements without any support from IDE. Approach with the interface looks more bulky:

```ts

interface BasePageElements {
  url: string;
}

interface LoginPageElements extends BasePageElements {
  enterAccountAddress: string;
  accountNameField: string;
  watchOnlyButton: string;
  polkadotVaultButton: string;
  continueButton: string;
  accessDeniedText: string;
  firstInfoButton: string;
  subscanLabel: string;
}

const LoginPageElements: LoginPageElements = {
  url: 'https://localhost:3000/#/onboarding',
  enterAccountAddress: 'Enter or paste your account address',
  accountNameField: 'Enter a name for your wallet',
  watchOnlyButton: 'text=Watch-only',
  polkadotVaultButton: 'text=Polkadot Vault',
  continueButton: 'Continue',
  accessDeniedText: 'text=Access denied!',
  firstInfoButton: '[id="headlessui-menu-button-\\:r4\\:"]',
  subscanLabel: 'View on Subscan',
};
```

Or you have better solution which I missed?

Reply:
Yeah, there's no right way in TS to declare an object with properties besides `BasePageElements` and have a nice type completion....
But using `class` just to store a bunch on properties isn't right as well
@Asmadek @egor0798 maybe you guys have some ideas how to add type completion for `LoginPageElements` not declaring additional interface?

Reply:
decided to live that approach as it is